### ISSUE TITLE
Unable to connect to extension marketplace on v 1.58

### ISSUE THREAD

**Issue by jurisbu** [VSCodium/vscodium#759]
**Describe the bug**
After upgrade to VS Codium 1.58 extension marketplace no longer works - cannot query or install extensions. Issue is with default open-vsx.org marketplace. No idea if changing to MS marketplace URL will work.

Downgrading to previous version 1.57.1 solves the issue. So, new bug in 1.58?

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

No idea, will not install VS Code on my main machine!

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
1. Go to View -> Extensions
2. Search for some extenson name from open-vsx.org
3. Error message: "We cannot connect to the Extensions Marketplace at this time, please try again later."

**Expected behavior**
Extensions marketplace can be searched and extensions installed.

**Screenshots**

**Desktop (please complete the following information):**
Version: 1.58.0
Commit: 2d23c42a936db1c7b3b06f918cde29561cc47cd6
Date: 2021-07-10T14:21:16.884Z
Electron: 12.0.13
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Linux x64 5.10.42-1-MANJARO

**Additional context**

From developer console:
```
Access to XMLHttpRequest at 'https://open-vsx.org/vscode/gallery/extensionquery' from origin 'vscode-file://vscode-app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
workbench.desktop.main.js:2835 Error querying extensions gallery XHR failed
query @ workbench.desktop.main.js:2835
async function (async)
query @ workbench.desktop.main.js:2835
(anonymous) @ workbench.desktop.main.js:2835
y @ workbench.desktop.main.js:68
show @ workbench.desktop.main.js:2835
show @ workbench.desktop.main.js:2835
(anonymous) @ workbench.desktop.main.js:2835
onDidAddViewDescriptors @ workbench.desktop.main.js:2835
(anonymous) @ workbench.desktop.main.js:2586
fire @ workbench.desktop.main.js:60
triggerOnDidAddVisibleViewDescriptors @ workbench.desktop.main.js:2351
onDidChangeContext @ workbench.desktop.main.js:2351
(anonymous) @ workbench.desktop.main.js:2351
fire @ workbench.desktop.main.js:60
(anonymous) @ workbench.desktop.main.js:58
fire @ workbench.desktop.main.js:60
fire @ workbench.desktop.main.js:60
setContext @ workbench.desktop.main.js:634
set @ workbench.desktop.main.js:634
updateInstalledExtensionsContexts @ workbench.desktop.main.js:2835
async function (async)
updateInstalledExtensionsContexts @ workbench.desktop.main.js:2835
create @ workbench.desktop.main.js:2835
create @ workbench.desktop.main.js:2586
showComposite @ workbench.desktop.main.js:2302
doOpenComposite @ workbench.desktop.main.js:2302
openComposite @ workbench.desktop.main.js:2302
doOpenViewlet @ workbench.desktop.main.js:2624
openViewlet @ workbench.desktop.main.js:2624
openViewContainer @ workbench.desktop.main.js:2624
run @ workbench.desktop.main.js:2624
handler @ workbench.desktop.main.js:652
invokeFunction @ workbench.desktop.main.js:625
_tryExecuteCommand @ workbench.desktop.main.js:1678
(anonymous) @ workbench.desktop.main.js:1678
Promise.then (async)
executeCommand @ workbench.desktop.main.js:1678
(anonymous) @ workbench.desktop.main.js:2772
emit @ events.js:315
onMessage @ electron/js2c/renderer_init.js:93
workbench.desktop.main.js:76 POST https://open-vsx.org/vscode/gallery/extensionquery net::ERR_FAILED
(anonymous) @ workbench.desktop.main.js:76
A @ workbench.desktop.main.js:76
request @ workbench.desktop.main.js:634
queryGallery @ workbench.desktop.main.js:639
async function (async)
queryGallery @ workbench.desktop.main.js:639
query @ workbench.desktop.main.js:639
(anonymous) @ workbench.desktop.main.js:2839
Promise.then (async)
queryGallery @ workbench.desktop.main.js:2839
queryGallery @ workbench.desktop.main.js:2835
query @ workbench.desktop.main.js:2835
(anonymous) @ workbench.desktop.main.js:2835
y @ workbench.desktop.main.js:68
show @ workbench.desktop.main.js:2835
show @ workbench.desktop.main.js:2835
(anonymous) @ workbench.desktop.main.js:2835
onDidAddViewDescriptors @ workbench.desktop.main.js:2835
(anonymous) @ workbench.desktop.main.js:2586
fire @ workbench.desktop.main.js:60
triggerOnDidAddVisibleViewDescriptors @ workbench.desktop.main.js:2351
onDidChangeContext @ workbench.desktop.main.js:2351
(anonymous) @ workbench.desktop.main.js:2351
fire @ workbench.desktop.main.js:60
(anonymous) @ workbench.desktop.main.js:58
fire @ workbench.desktop.main.js:60
fire @ workbench.desktop.main.js:60
setContext @ workbench.desktop.main.js:634
set @ workbench.desktop.main.js:634
updateInstalledExtensionsContexts @ workbench.desktop.main.js:2835
async function (async)
updateInstalledExtensionsContexts @ workbench.desktop.main.js:2835
create @ workbench.desktop.main.js:2835
create @ workbench.desktop.main.js:2586
showComposite @ workbench.desktop.main.js:2302
doOpenComposite @ workbench.desktop.main.js:2302
openComposite @ workbench.desktop.main.js:2302
doOpenViewlet @ workbench.desktop.main.js:2624
openViewlet @ workbench.desktop.main.js:2624
openViewContainer @ workbench.desktop.main.js:2624
run @ workbench.desktop.main.js:2624
handler @ workbench.desktop.main.js:652
invokeFunction @ workbench.desktop.main.js:625
_tryExecuteCommand @ workbench.desktop.main.js:1678
(anonymous) @ workbench.desktop.main.js:1678
Promise.then (async)
executeCommand @ workbench.desktop.main.js:1678
(anonymous) @ workbench.desktop.main.js:2772
emit @ events.js:315
onMessage @ electron/js2c/renderer_init.js:93
DevTools failed to load SourceMap: Could not load content for https://ticino.blob.core.windows.net/sourcemaps/2d23c42a936db1c7b3b06f918cde29561cc47cd6/core/vs/workbench/workbench.desktop.main.js.map: Load canceled due to load timeout
```

**Comment by daiyam** [comment]
Hi, it's a duplicate of #746.
